Rakefile

```diff
@@ -286,9 +323,20 @@ def each_gem_dir
 
     yield subdir
   end
+
+  Dir.chdir("#{__dir__}/data")
```

Comment:
similar to above comments, could we parameterize this code instead of copy/pasting?